File: DheerajKumar/problem09.cpp
//9. WAP to check whether a number is even or odd.

#include<iostream>
using namespace std;

bool isEven(int number){
    if(number % 2 == 0){
        return true;
    }
    return false;
}

int main(){
     
    int number;
    cout<<"\n\nenter the numebr ->>";
    cin>> number;

    if(isEven(number)){
        cout<<"\n < EVEN >";
    }else{
        cout<<" \n < ODD >";
    }

}

File: LovelyKumari/BasicProgramming/Problem09.java
import java.util.Scanner;

public class Problem09 {
    //WAP to check whether a number is even or odd.
    public static void main(String[] args) {
        Scanner sc= new Scanner(System.in);
        System.out.print("Enter a number : ");
        int num = sc.nextInt();

        //Check if even or odd 
        if(num % 2 == 0 ){
            System.out.println(num + " is an even number.");
        } else {
            System.out.println(num + " is a odd number.");
        }

        
    }

    
}
